cde-core/resource/resources/custom/components/exportButton/component.xml

```diff
@@ -90,6 +91,20 @@
             <Version>1.0</Version>
           </Header>
         </DesignerProperty>
+        <DesignerProperty>
+          <Header>
+            <Name>exportPage</Name>
+            <Parent>BaseProperty</Parent>
+            <DefaultValue>false</DefaultValue>
+            <Description>Export Page</Description>
+            <Tooltip>Export single page, only relevant for server side paginated components</Tooltip>
+            <InputType>Boolean</InputType>
+            <OutputType>Boolean</OutputType>
+            <Order>51</Order>
+            <Advanced>false</Advanced>
```

Comment:
We might prefer to have this property as Advanced actually, I would change this to **true**